viewer/src/tools/AxisView/AxisViewTool.ts

```diff
@@ -0,0 +1,321 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as THREE from 'three';
+import ComboControls from '@cognite/three-combo-controls';
+import TWEEN from '@tweenjs/tween.js';
+import _ from 'lodash';
+
+import { Cognite3DViewer } from '../../migration';
+import { Cognite3DViewerToolBase } from '../Cognite3DViewerToolBase';
+import { AxisBoxConfig, defaultAxisBoxConfig, AxisBoxFaceConfig, Corner, Absolute, Relative } from './types';
+
+export class AxisViewTool extends Cognite3DViewerToolBase {
+  private readonly _layoutConfig: AxisBoxConfig;
```

Comment:
```suggestion
  private readonly _layoutConfig: Required<AxisBoxConfig>;
```
https://www.typescriptlang.org/docs/handbook/utility-types.html#requiredtype

By doing this you can avoid some `!` in the code below.